### PR TITLE
Promote use of opam local switch

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: benchmark.md [hunk 1]
```diff
@@ -11,20 +11,18 @@
 ### OCaml 4.13.1
 
 ```sh
-opam switch 4.13.1
-eval $(opam env)
-dune build --profile=release
-dune exec bin/sdl2/bench.exe -- resource/games/tobu.gb --frames 1500
+opam switch . ocaml-base-compiler.4.13.1
```
Comment (linoscope) [inline benchmark.md:14]: I get the following error message when I run this command:
```
opam: Invalid switch subcommand "."
Usage: opam switch [OPTION]... [COMMAND] [ARG]...
Try `opam switch --help' or `opam --help' for more information.
```
Any idea on why this is happening? I tried with both opam version `2.0.5` and `2.1.2`.

Comment (smorimoto) [reply benchmark.md:14]: Oops... I meant `opam switch create . ocaml-base-compiler.4.13.1`! But not really!

**Review by linoscope — APPROVED** [review]
LGTM, thanks!

**Comment by smorimoto** [comment]
It's always recommended to work with the opam local switch!

**Comment by linoscope** [comment]
@smorimoto Good to know! I am still pretty new to opam/dune and the interaction between them so learning these best practices is helpful, thanks!

**Pull request merged**